rpc/src/v1/helpers/errors.rs

```diff
@@ -173,11 +173,11 @@ pub fn state_corrupt() -> Error {
 	internal("State corrupt", "")
 }
 
-pub fn exceptional() -> Error {
+pub fn exceptional<T: fmt::Debug>(data: T) -> Error {
```

Comment:
I think `fmt::Display` implementation for `vm::Error` is more user-friendly.

Reply:
I'm I understand correct that if we switch to `fmt::Display` this formatter will be used?
https://github.com/paritytech/parity-ethereum/blob/9cb86061032a323710a1dd549e1e1f0a4e33eedd/ethcore/vm/src/error.rs#L100-L117

Reply:
Yes, that's my suggestion, but I don't have a strong opinion about that.